uri-schemes.md

```diff
@@ -124,6 +126,88 @@ HTTP Status code: NOT 2XX, idealy the 4XX if it's user error and 5XX if it's the
 
 json object can have other properties too, but currently they are ignored by core.
 
+## **DCLOGIN** <a name="DCLOGIN"></a>
+
+|                          |                       |
+| ------------------------ | --------------------- |
+| Scheme                   | `DCLOGIN:`            |
+| Used for                 | Account setup         |
+| Related Terms\*          | Account Login QR code |
+| Available on             | draft                 |
+| Decoded by the core \*\* | draft                 |
+| Other apps using it      | none, only DeltaChat  |
+
+### Syntax
+
+```
+dclogin://user:password@host/?options
+dclogin://user:password@host?options
+dclogin:user:password@host/?options
+# example: (email: me@example.com, password: securePassword)
+dclogin://me:securePassword@example.com?v=1
```

Comment:
not sure if it is a thing but IIRC some servers use only the local part of the email address as the username to log in, how would that be specified here? or this is only for servers that the login is with the email address and that the server address ends with @host (and host is the right imap/smtp host?

personally would have preferred something like dclogin://addr:password?mail_server=...&send_server=...&(other dc config options here)

Reply:
and then calling it `dcsetup:` would makes more sense since you could setup several config options for the user, would be more useful for admins setting up servers for easy onboarding